web/api/backend.go

```diff
@@ -92,8 +92,8 @@ func wrapperExecutor(wc WrapperCommand) string {
 // createCredentialsToken takes as input a representiation of the uplodaded uploadCredentials json body and returns a token using the
 // uploadCredentials json body as the payload.
 func createCredentialsToken(Credentials map[string]interface{}) (string, error) {
-	// Declare the expiration time of the token.
-	expirationTime := time.Now().Add(tokenExpiryMinutes * time.Minute)
+	// Declare the expiration time of the token. Here, we have kept it as 1 day.
+	expirationTime := time.Now().Add(1440 * time.Minute)
```

Comment:
How come here the you changed the value of tokenExpiryMinutes to be 1440? I think Andy had mentioned that we have to restrain from using literal values, so ideally we would keep the variable.